src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -805,8 +841,67 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
     </div>
   );
 
+  const { sovereignVenueId } = useSovereignVenueId({ venueId });
+
+  const { currentVenue: venue } = useConnectCurrentVenueNG(sovereignVenueId);
```

Comment:
can we rename it to `sovereignVenue` not to get confused?

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/71f5181e9d01672881376d7c0a0f0f0f917511eb